tests/test_activate.py

```diff
@@ -1105,40 +1107,292 @@ def make_dot_d_files(prefix, extension):
     touch(join(prefix, "etc", "conda", "deactivate.d", f"deactivate1{extension}"))
 
 
+@pytest.mark.skipif(
+    not on_win,
```

Comment:
Shouldn't this be run outside of Windows then to make sure that this is the case?

Reply:
On Windows the `native_path_to_unix` and `unix_path_to_native` functions will short-circuit to the `path_identity` function instead which won't error, they will simply return gibberish for the input values.